src/utils/formHelperFunctions.js

```diff
@@ -52,14 +48,14 @@ const formatStepFields = (step, fieldValues, signatureRef) => {
 
     const servar = field.servar;
     let value;
-    if (servar.type === 'signature') {
+    // Only use base64 for signature if these values will be presented to the user
+    if (servar.type === 'signature' && forUser) {
       value =
-        signatureRef && signatureRef[servar.key]
-          ? dataURLToFile(
-              signatureRef[servar.key].toDataURL('image/png'),
-              `${servar.key}.png`
+        fieldValues[servar.key] !== ''
```

Comment:
this ternary may be cleaner if you first store
const val = fieldValues[servar.key];

not confident about that tho, feel free to disregard

Reply:
also seems like the ternary can go within the Promise.resolve() rather than outside it